cubic-server/Dimension.hpp

```diff
@@ -15,6 +16,11 @@
 
 // TODO(huntears): Fix whatever this is
 constexpr int SEMAPHORE_MAX = 1000;
+constexpr int TICK_PER_MINUTE = 20 * 60;
+constexpr int TICKS_FOR_FIVE_MINUTES = TICK_PER_MINUTE * 5;
+constexpr int TICKS_FOR_FIFTEEN_MINUTES = TICK_PER_MINUTE * 15;
```

Comment:
A simple calculation like this don't need to be constexpr, just use `TICK_PER_MINUTES`